gpu_serving/object-detection-app/components/params.libsonnet

```diff
@@ -0,0 +1,16 @@
+{
+  global: {},
+  components: {
+    // Component-level parameters, defined initially from 'ks prototype use ...'
+    // Each object below should correspond to a component in the components/ directory
+    model1: {
+      cloud: 'gcp',
+      deployHttpProxy: true,
+      gcpCredentialSecretName: 'kai-sa',
```

Comment:
Why are you using the GCP secret kai-sa? I believe the GKE deploy secrets will create the secret user-sa does that work?

Reply:
I was reading  model in my bucket. user-sa should work.

Reply:
Can you change the default parameters to be parameters that will work for users here and below.